From 77109fd5481a622121a9ca20e19d76fc97125e47 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Wed, 17 May 2023 11:16:56 +0200
Subject: [PATCH] man/tmpfiles: fix off-by-one in example

Reported and diagnosed by gitterman. Fixes #26617.

(cherry picked from commit f90360eb7417e083650034ad819790df0c389bd1)

Resolves: RHEL-74015
---
 man/tmpfiles.d.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/man/tmpfiles.d.xml b/man/tmpfiles.d.xml
index fe2a1dadab..379ac9c34f 100644
--- a/man/tmpfiles.d.xml
+++ b/man/tmpfiles.d.xml
@@ -644,7 +644,7 @@ w- /proc/sys/vm/swappiness - - - - 10</programlisting></para>
       <para>For example:<programlisting>
 # Files created and modified, and directories accessed more than
 # an hour ago in "/tmp/foo/bar", are subject to time-based cleanup.
-d /tmp/foo/bar - - - - bmA:1h -</programlisting></para>
+d /tmp/foo/bar - - - bmA:1h -</programlisting></para>
 
       <para>Note that while the aging algorithm is run a 'shared' BSD file lock (see <citerefentry
       project='man-pages'><refentrytitle>flock</refentrytitle><manvolnum>2</manvolnum></citerefentry>) is
